boards/common/arduino-atmega/include/periph_conf.h

```diff
@@ -61,6 +63,17 @@ extern "C" {
 #define TIMER_0_ISRB        TIMER1_COMPB_vect
 #endif
 
+#ifdef CPU_ATMEGA32U4
+#define TIMER_NUMOF         (2U)
```

Comment:
`TIMER_CHANNELS` is missing (most errors I've seen so far on Murdock can be traced to this).

Reply:
(also you state here that `TIMER_NUMOF` is `2U`, but there is only the config for one timer provided).

Reply:
I pushed a fix for that.